src/io/Display.cpp

```diff
@@ -69,23 +64,22 @@ namespace chip8::io
         for (unsigned int i = 0; i < displayPixels_.kWidth_; ++i)
             for (unsigned int j = 0; j < displayPixels_.kHeight_; ++j)
```

Comment:
This is the kind of loop that feels better with a std algorithm and iterators.

Reply:
It's copy pasted code. Don't want to lose too much time with the SFML implementation for the moment.

Reply:
I just modified so instead of the attached image, it would paint the pixel values:
![image](https://user-images.githubusercontent.com/3214400/58796954-f16ab380-85fe-11e9-825f-81b001e8d183.png)